Taskfile.yml

```
@@ -1,88 +1,88 @@
-      - cd apps/indexer && bun preapack
-
-  indexer:dev:
-    desc: "Run the Subquery Indexer in development mode"
-    cmds:
-      - cd apps/indexer && bun dev
-
-  indexer:start-docker:
-    desc: "Start the Subquery Indexer in a Docker container with an indexer, PostgeSQL DB, and a query service"
-    cmds:
-      - cd apps/indexer && bun start:docker
+version: "3"
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)